resources/knative-provisioner-natss/charts/knative-provisioner-natss/templates/natss.yaml

```diff
@@ -1,430 +1,428 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-    labels:
-        contrib.eventing.knative.dev/release: "v0.10.0"
-        duck.knative.dev/addressable: "true"
-    name: natss-addressable-resolver
+  labels:
+    contrib.eventing.knative.dev/release: "v0.12.0"
```

Comment:
(nit): quotes are optional

Reply:
I tried to take the charts as unmodified as possible to reduce future diffs. So I would like to keep it as it is. But thanks for pointing it out :)